File: Resueltos_por_competidor/Juan_Martinez/Koshary.cpp
/*
 * Autor: Juan Martinez
 * Problema: Koshary (2227A)
 * Juez online: Codeforces
 * Veredicto: Accepted
 * Url: https://codeforces.com/contest/2227/problem/A
 */

#include <bits/stdc++.h>

using namespace std;

int main()
{
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    int n, x, y;
    cin>>n;
    while(n--) {
        cin>>x>>y;
        if((x%2 != 0) && (y%2 != 0)) cout<<"NO"<<endl;
        else  cout <<"YES"<< endl;
    }

    return 0;
}

File: Resueltos_por_competidor/Jairo_Esteban/Koshary.java
import java.util.Scanner;

/*
 * Autor: [kjairoo]
 * Problema: [Koshary]
 * Juez online: [Codeforce]
 * Veredicto: Accepted
 * URL: https://codeforces.com/problemset/problem/2227/A
*/

public class Koshary {

    static Scanner sc = new Scanner(System.in);

    public static void main(String[] args) {

        int T = sc.nextInt();

        while (T-- > 0){

            int X = sc.nextInt();
            int Y = sc.nextInt();

            if ((X % 2 == 1) && (Y % 2 == 1)){
                System.out.println("NO");
            } else {
                System.out.println("YES");
            }
        }
    }

}
